common/src/main/java/org/conscrypt/OpenSSLProvider.java

```diff
@@ -50,6 +50,10 @@ public OpenSSLProvider() {
     }
 
     public OpenSSLProvider(String providerName) {
+        this(providerName, false);
```

Comment:
should the default be true?

Reply:
No, that would mean people get opted into the new TrustManager, and I don't think we want that.